supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -145,20 +145,25 @@ class JesAsyncBackendJobExecutionActor(override val standardParams: StandardAsyn
     }
   }
 
-  private[jes] def generateJesInputs(jobDescriptor: BackendJobDescriptor): Set[JesInput] = {
+  private[jes] def generateJesInputs(jobDescriptor: BackendJobDescriptor): Try[Set[JesInput]] = {
 
-    val writeFunctionFiles = call.task.evaluateFilesFromCommand(jobDescriptor.fullyQualifiedInputs, backendEngineFunctions) map {
-      case (expression, file) =>  expression.toWdlString.md5SumShort -> Seq(file)
-    }
+    commandLinePreProcessor(jobDescriptor.inputDeclarations) map { preprocessedInputs =>
```

Comment:
I think you're correct. I went belt-and-braces by applying both preProcessor and the valueMapper but for JES only the valueMapper actually does anything.

Good spot!